Sample.java

```diff
@@ -1,7 +1,102 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : O(N) - contains logic require atmost N iterations to check if the element is present.
+// Space Complexity : O(N) - though array size is fixed, the list in each array grows with the number of elements (N)
+// Did this code successfully run on Leetcode : yes
+// Any problem you faced while coding this : None
 
 
 // Your code here along with comments explaining your approach
+
+import java.util.Iterator;
+import java.util.LinkedList;
+// A simple approach without using any hash function. 
+// I am just using an arraylist to store the elements and everytime
+// I am checking if it is present in the list before adding or removing it.
+// To check if the element is present, I am using indexOf method. 
+// It worked on LeetCode.
+// TC: O(N) for all methods.
+/*class MyHashSet { 
+    ArrayList<Integer> set;
+    public MyHashSet() {
+        set = new ArrayList<>();
+    }
+    
+    public void add(int key) {
+        if(contains(key)) {
+            return;
+        }
+        set.add(key);
+    }
+    
+    public void remove(int key) {
+        if(contains(key)) {
+            int idx = set.indexOf(key);
+            set.remove(idx);
+        }
+    }
+    
+    public boolean contains(int key) {
+        return set.indexOf(key) != -1;
+    }
+}*/
+
+
+/**
+ * @author akhilreddy619
+ * In this class, I am using an array of linkedlist.
+ * _hash: (extra function added) to calculate the index in which the element has 
+ * to be inserted/removed.
+ * For Add: calculating the hash and check if it has a list in that index. If so,
+ * check if the key is already inserted, if not insert it else skip it. If the list
+ * is not initialized, then initialize it and insert the key. Then put that list in 
+ * the array.
+ * For Remove:Calcuate the hash and check if it has a list in that index. If so, 
+ * iterate that list and remove the element else skip it.
+ * For Contains: Calculate the hash and check if it has a list in that index. If not,
+ * return false, else check if that list has this key and return boolean accordingly.
+ *
+ */
```

Comment:
Good Job. Mention time and space complexity.

Reply:
TC and SC are mentioned at the top of the file.

Reply:
Time complexity to access element in hashset is O(1)

Reply:
Ideally yes, but I mentioned TC based on the code I wrote. In each method, I am either using contains or iterating over the elements in the list to remove which is an O(n) operation. Thus, I mentioned TC as O(n)

Reply:
Okay, please write the optimal solution also.

Reply:
added the new optimal solution.